ordered_associative_task/student/include/impl.h

```diff
@@ -15,4 +15,18 @@
  *
  * @return 1-5
  */
-size_t convertMarkTo5PointSystem(size_t markIn100Point);
+
+std::map<size_t, size_t> gradePoints = {
+    {29, 1},
+    {49, 2},
+    {69, 3},
+    {89, 4},
+    {100, 5}
+};
+
+size_t convertMarkTo5PointSystem(size_t markIn100Point)
+{
+    auto it = gradePoints.lower_bound(markIn100Point > 100 ? 100 : markIn100Point);
```

Comment:
hacky way, try to solve it with such mapping
{{0, 1}, {30, 2}, {50, 3}, {70, 4}, {90, 5}};

Reply:
Can you give me a hint on how I should solve this, please? I tried many solutions, but can only work for only one mapping.